engines/cge/talk.cpp

```diff
@@ -88,6 +91,15 @@ Talk::Talk(CGEEngine *vm)
 	_wideSpace = false;
 }
 
+const char *lastText = "";
+
+void textToSpeech(const char *text) {
+	Common::TextToSpeechManager *ttsMan = g_system->getTextToSpeechManager();
+	if (lastText != text && ttsMan != nullptr && ConfMan.getBool("tts_enabled"))
+		ttsMan->say(text, Common::TextToSpeechManager::INTERRUPT);
+		lastText = text;
+}
```

Comment:
Do you need tp check `lastText` here? Wouldn't using `INTERRUPT_NO_REPEAT` work just as well?

Reply:
I followed all the reviews you did, except for this one, I've only fixed the wrong indentation and removed `INTERRUPT_NO_REPEAT`, because when that action item is used independently, the text repeats itself at the beginner of an utterance.

Reply:
Yes, as you pointed out on the cge2 pull request, the `INTERRUPT_NO_REPEAT` is indeed not sufficient. So you can ignore this comment.

Reply:
Also you have missing brackets here, but this has already been fixed in master by digitall in e821511050.